src/models/flinkUDF.test.ts

```diff
@@ -5,33 +5,7 @@ import { ConnectionType } from "../clients/sidecar";
 import { CCLOUD_CONNECTION_ID } from "../constants";
 import { FlinkUdfParameter, FlinkUdfTreeItem, createFlinkUdfToolTip } from "./flinkUDF";
 
-describe("flinkUDF.ts", () => {
-  describe("FlinkUdfParameter", () => {
-    describe("formatSqlType", () => {
```

Comment:
This external function / its test now moved over into `src/utils/flinkTypes[.test].ts`. The function is also used by `src/models/flinkSystemCatalog.ts`